content/en/cosign/openid_signing.md

```diff
@@ -4,71 +4,40 @@ category: "Cosign"
-Certificate subject:  user@google.com
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36"},"Type":"cosign container image signature"},"Optional":null}
-```
+Keyless signing uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
 
-That's it! No keys!
-The rest of the flags (annotations, claims, tlog, etc.) should all work the same.
+Information on the Fulcio root CA can be found in the [fulcio repository](https://github.com/sigstore/fulcio).
 
-## Overview
+## Keys
 
-This uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
+The root CA keys leverage the principles of The Update Framework (TUF).  TUF is a set of defined attacks and threat models specific to software distribution systems. It provides a set of protocols to protect against various types of attacks, which the Trust Root follows. For more information about how Sigstore uses TUF, see [How Sigstore uses TUF](https://dlorenc.medium.com/using-the-update-framework-in-sigstore-dc393cfe6b52)
 
-Information on the Fulcio root CA can be found in the [fulcio repository](https://github.com/sigstore/fulcio).
+## OAuth flows
 
-### Keys
+Cosign supports two OAuth flows today: the standard flow and the device flow.
 
-The root CA keys are hard-coded in Cosign today. They can only be changed by recompiling the binary. This will be made more configurable in the future.
+When there is no terminal attached (non-interactive mode), Cosign will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
 
-### OAuth flows
+## Identity tokens
 
-Cosign supports two OAuth flows today: the standard flow and the device flow.
+In automated environments, Cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` field must contain `sigstore`.
 
-When there is no terminal attached (non-interactive mode), `cosign` will automatically use the device flow where a link is printed to stdout. This link must be opened in a browser to complete the flow.
+### Logging in to identity issuers
 
-### Identity tokens
+To log in and set up your OIDC Identity, follow these steps:
 
-In automated environments, cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` field must contain `sigstore`.
+1. Navigate to the [Issuer login page](https://oauth2.sigstore.dev/auth/auth).
```

Comment:
```suggestion
1. Follow the link that Cosign provides to the Issuer login page.
```

Reply:
The link has more parameters, specific to a single login, so Id rather say something more generic